terraform/modules/certificates/certificates.tf

```diff
@@ -2,6 +2,8 @@ resource "aws_acm_certificate" "cloudfront" {
   domain_name       = var.cloudfront_domain_name
   validation_method = "DNS"
 
+  subject_alternative_names = var.cloudfront_additional_names
```

Comment:
Cloudfront only allows one certificate attached at once, so will need a cert for both in order to do the switchover.

You can attach multiple certificates to one load balancer though.